.github/ISSUE_TEMPLATE/avm_module_issue.yml

```diff
@@ -44,6 +44,7 @@ body:
         # - "avm/ptn/avd-lza/session-hosts"
         # - "avm/ptn/security/security-center"
         # - "avm/res/aad/domain-service"
+        - "avm/ptn/authorization/role-assignments"
```

Comment:
This should be placed in alphabetical order. Static validation should fail on this. Isn't that the case?

Reply:
Checked and fixed